JDI.Light/JDI.Light/Elements/Composite/MultiDropdown.cs

```diff
@@ -0,0 +1,138 @@
+﻿using JDI.Light.Elements.Base;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using JDI.Light.Attributes;
+using JDI.Light.Elements.Common;
+
+namespace JDI.Light.Elements.Composite
+{
+    public class MultiDropdown : UIElement
+    {
+        [FindBy(Tag = "button")]
+        private Button _header;
+        
+        public List<MultiDropdownElement> Options
+        {
+            get
+            {
+                var elems = FindElements(By.XPath(".//li")).ToList();
+                var labels = FindElements(By.XPath(".//li//label")).ToList();
+                var chBoxes = FindElements(By.XPath(".//li//input")).ToList();
+                List<MultiDropdownElement> elementList = new List<MultiDropdownElement>();
+                for (int i = 0; i < elems.Count; i++)
+                {
+                    elementList.Add(new MultiDropdownElement(By.XPath(".//li"),labels[i],chBoxes[i], elems[i]));
+                }                
+                return elementList;
+            }
+        }
+
+        public bool OptionIsEnabled(string name)
+        {            
+            return Options.Any(x => x.OptionIsEnabled && x.Text == name);
+        }
+
+        public void SelectOptionByname(string name)
```

Comment:
ByName instead of Byname:
![image](https://user-images.githubusercontent.com/6689884/54746994-ab48ab00-4bde-11e9-835c-bcda56c1ef8d.png)